src/main/java/ui/UI.java

```diff
@@ -143,6 +143,14 @@ private void login(boolean isBypassLogin) {
         getMainWindowHandle(mainStage.getTitle());
     }
 
+    private void launchSampleBoard(){
+        BoardAutoCreator boardCreator = new BoardAutoCreator(this, panels, prefs);
+        if (prefs.getAllBoardNames().contains(boardCreator.SAMPLE_BOARD)){
+            prefs.removeBoard(boardCreator.SAMPLE_BOARD);
```

Comment:
Maybe you could explain why this part is needed with an example in a comment? Is there a case where it could remove something legitimate?